x-pack/plugins/data_visualizer/public/application/index_data_visualizer/utils/get_supported_aggs.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { DataViewField } from '@kbn/data-views-plugin/common';
+
+export const isCounterTimeSeriesMetricField = (field: DataViewField) =>
+  field.timeSeriesMetric === 'counter';
+
+const SUPPORTED_AGGS = {
+  COUNTER: new Set(['count', 'min', 'max']),
```

Comment:
For `counter` fields the action to Lens is still using `avg` and so no chart is displayed in Lens on opening. Could the action  use `max` for counter fields?

Reply:
Great call! Updated here [`8e7f88f` (#153893)](https://github.com/elastic/kibana/pull/153893/commits/8e7f88fdd838384a40a80e9667276dc4bba3a56a)